addons/digest/models/res_users.py

```diff
@@ -6,13 +6,14 @@
 class ResUsers(models.Model):
     _inherit = "res.users"
 
-    @api.model
-    def create(self, vals):
+    @api.model_create_multi
+    def create(self, vals_list):
         """ Automatically subscribe employee users to default digest if activated """
-        user = super(ResUsers, self).create(vals)
+        users = super(ResUsers, self).create(vals_list)
         default_digest_emails = self.env['ir.config_parameter'].sudo().get_param('digest.default_digest_emails')
         default_digest_id = self.env['ir.config_parameter'].sudo().get_param('digest.default_digest_id')
-        if user.has_group('base.group_user') and default_digest_emails and default_digest_id:
```

Comment:
the advantage of this one is that it was ormcached (which be a disadvantage in case of mass-import...), have you compared the performances with `filtered_domain`?

Reply:
`has_group` is ormcached yes, but filling the cache triggers one query by user (we've just created the users, there is nothing in the cache for them atm).  

IMO, pre-filling the cache when we don't even know when those users will be used (and their groups verified by another has_group) is not efficient at all.

If you consider it is worth to prefill the ormcache of has_group for all newly created users, then I should filter user on the `share` fields to know the ones which should be added to the default digest (btw, if we do not consider it as worth it, I should also refactor the `_compute_share` method to avoid doing all those `has_group` at flush)

`filtered_domain`, on the other hand, will use the values in cache (after the create, the information is expected to be in cache anyway).